mis_builder_total_committed_purchase/__manifest__.py

```diff
@@ -6,14 +6,14 @@
     "name": "MIS Builder Total Committed Purchase",
     "summary": """Addon to create a alternative source based on all purchase order line with
     MIS Builder.""",
-    "version": "15.0.1.0.1",
+    "version": "16.0.1.0.0",
     "license": "AGPL-3",
     "author": "ACSONE SA/NV, Camptocamp, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/mis-builder-contrib",
     "depends": ["mis_builder", "purchase"],
     "data": [
         "data/mis_total_committed_purchase.sql",
-        "data/mis_total_committed_purchase_tag_rel.sql",
+        "data/mis_total_committed_purchase_analytic_account_rel.sql",
         "security/mis_total_committed_purchase.xml",
         "views/mis_total_committed_purchase.xml",
```

Comment:
Do we need access this view on UI ?

Reply:
yes, in order to view the source data

Reply:
I mean is there any way to access this view on UI ?

Reply:
you can install the module `mis_builder_demo`, then changing the source of one label, then you can drilldown the data which is pointing to proper data source

![image](https://user-images.githubusercontent.com/46039081/224684016-bb6bbef1-651b-4d42-8cd4-2f45702e933a.png)